lib/avromatic/schema_registry_patch.rb

```diff
@@ -7,24 +7,26 @@ module CacheableSchemaRegistration
     def register(subject, schema)
       return super unless Avromatic.use_cacheable_schema_registration
 
+      begin
+        lookup_subject_schema(subject, schema)
+      rescue
```

Comment:
Wouldn't a server that doesn't support this endpoint return a 404? As written it seems like we'll mask bugs in this code.